spec/models/user_spec.rb

```diff
@@ -208,4 +208,48 @@
       end
     end
   end
+
+  describe '#avatar_format' do
```

Comment:
I wouldn't add these tests as tests for a private method, but in the `#valid?` block.

Reply:
Done. ✅